users/permissions.py

```diff
@@ -0,0 +1,113 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+from . import WINERY
+from api.models import Winery
+
+SAFE_ACTIONS = ['list', 'retrieve']
+NOT_SAFE_ACTIONS = ['create', 'update', 'partial_update', 'destroy']
+
```

Comment:
Some context for the review:
This part is really important, we should be careful. 
has_permission is called always, and if it returns true, the method has_object_permission is called after with a specific object as a parameter. (So it shouldn't be called on the list method).